tagstudio/src/qt/ts_qt.py

```diff
@@ -238,9 +241,6 @@ def start(self):
 
         # Handle OS signals
         self.setup_signals()
-        timer = QTimer()
-        timer.start(500)
-        timer.timeout.connect(lambda: None)
```

Comment:
The removal of the timer here stops the Qt thread from pausing to process console input, therefore breaking the ability to handle SIGTERMs sent from the console.

Reply:
Ah that's what this is for. I thought it's some leftover code doing nothing. I added a comment there to make it more obvious